src/PowerSimulations.jl

```diff
@@ -590,5 +591,6 @@ include("utils/jump_utils.jl")
 include("utils/powersystems_utils.jl")
 include("utils/recorder_events.jl")
 include("utils/datetime_utils.jl")
+include("utils/generate_valid_formulations.jl")
```

Comment:
I did not export any of the new functions because I don't expect users to call them. Let me know if you disagree.